README.md

```diff
@@ -0,0 +1,28 @@
+# CoMPAS OpenSCD Document Operations Monorepo
+
+This repository hosts the components and plugins related to the (db-stored) SCL documents lifecycle management within the CoMPAS OpenSCD project.
+
+## Components and Plugins
+
+The following components are featured in this repository:
+
+- `compas-loading` (**TODO**: add README to compas-loading)
```

Comment:
Make sure we have an issue for the open TODO items; It makes sense to keep track of all the work that needs to be done.

Reply:
https://github.com/com-pas/compas-oscd-docops/issues/19